packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -149,7 +149,8 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
       display: isMobileWithActivePane ? 'none' : 'flex',
       minWidth: props.mobileView ? 'unset' : `${compositeMinWidthRem}rem`,
       width: '100%',
-      height: '100%'
+      height: '100%',
+      position: 'relative'
```

Comment:
nice fix! Can we add an e2e test for this? It would be very easy for someone to refactor this and accidental regress